packages/itwin/measure-tools/src/tools/MeasureAreaTool.ts

```diff
@@ -35,6 +35,8 @@ MeasureAreaToolModel
   public static override toolId = "MeasureTools.MeasureArea";
   public static override iconSpec = "icon-measure-2d";
   private _enableSheetMeasurements: boolean;
+  private _drawingTypeCache: SheetMeasurementsHelper.DrawingTypeData[] = [];
```

Comment:
Please create a dedicated cache object and reuse it across tools instead of copying the cache code multiple times.

Reply:
@a-gagnon Meaning adding a property to the measurementToolBase class instead?

Reply:
I was thinking of a separate class implementation -- having this in the base class might not be desirable if someone writes specialized measurements like the section anchored measurements. We wouldn't need/care for sheet information in those cases.

Reply:
Sounds good!